querydsl-sql/src/main/java/com/querydsl/sql/SQLListeners.java

```diff
@@ -245,4 +245,9 @@ public void exception(final SQLListenerContext context) {
             listener.exception(context);
         }
     }
+
+    public Set<SQLDetailedListener> getListeners() {
```

Comment:
I like the emphasis on no duplicates instead of ordering, although a LinkedHashSet is used